### DISCUSSION TITLE
Error looking up 'colima-Darwin-arm64'

### DISCUSSION THREAD

**Discussion by mab** [abiosoft/colima#1117] [Q&A] [answered]
Hello,

I use [aqua](https://aquaproj.github.io/) to install colima on MacOS 14 with an Mx chip. Aqua has a proxy to redirect to the correct binary. This works good though I get an warning that `colima-Darwin-arm64` is not on my path -- what is correct. 

```
colima version v0.7.3
git commit: a66b375e8df84ff2860797efc683e66632bcbce3
WARN[0000] cannot detect current running executable: error looking up 'colima-Darwin-arm64' in PATH: exec: "colima-Darwin-arm64": executable file not found in $PATH
WARN[0000] falling back to first CLI argument
```

Why do I get the warning? What do I need to do in order to get rid of it?

Matthias

**Comment by abiosoft** [comment, marked as answer]
Colima actually re-launches itself in background as a daemon to provide some additional functionalities.
In the process, it detects the full path of the current running executable.

I suspect the way aqua works is a bit unconventional and confuses the detection.

Regardless, the warning is harmless and can be discarded as it simply falls back to `$0` and that works fine.

As a fix, an update can be pushed to lower the log level of the message to trace so it does not show up normally.

**Reply by mab** [reply]
Got it. Then I continue to ignore the warning. :)
